### DISCUSSION TITLE
阿茲海默症

### DISCUSSION THREAD

**Discussion by Terryyyyyyyyyyyy** [1101-GS4719-C/Python-Community-GS4719-C#1855] [General]
我最近刷影片時刷到有一位阿茲海默症患者，同時也是位視障者，在得到阿茲海默症後期，因為忘記他眼睛看不到，竟然重獲光明，想問大家認為這件事情是真的嗎？
我認為有可能是假的，因為整個影片上下透出一種造假的方式，而且如果真有可能因為這樣重獲光明，那應該已經上了醫學期刊了

**Comment by Terryyyyyyyyyyyy** [comment]
我還看到有阿茲海默症患者因為忘記自己失憶，反而記起所有事情，這看起來也像假的，因為會忘記自己忘記了甚麼，就像已經及其所有事情了，有可能是傳出來騙眼球，跟點閱的

**Comment by asd99263fg** [comment]
為了避免阿茲海默症，可以保持健康的生活方式，包括均衡飲食、定期運動和充足睡眠；保持心智活躍，通過閱讀、學習新技能和參與社交活動來刺激大腦；管理好心血管健康，控制血壓、血糖和膽固醇水平；避免吸菸和過量飲酒，並定期進行健康檢查以及早發現和處理健康問題。這些綜合性的預防措施有助於降低患阿茲海默症的風險。

**Comment by lilishh** [comment]
罹患阿茲海默症與視障疾病，應該沒有相關聯？所以患上阿茲海默症後，重獲光明真的不太可能，如果屬實，應該醫學界已經在研究可以如何是好視障了吧

>
